### DISCUSSION TITLE
kairship/workload/job

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2365] [Announcements]
# kairship/workload/job

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/workload/job.html

**Comment by yt-huang** [comment]
没有yaml文件创建job的文档

**Reply by windsonsea** [reply]
Fixed in PR #5309
